test/test_main.py

```diff
@@ -130,6 +133,8 @@ def execute_test_case_for_transformation(transformation_name):
             execute_tagging_test_case(transformation, test)
         else:
             print(f"Invalid transformation type: {transformation}")
+        # Reinitialize spacy tokenizer [TODO: Need to run only for transformations using spacy]
+        reinitialize_spacy()
```

Comment:
Just to verify, can we just try keeping this reinitialize function here and remove it from the constructor of the transformations? As here will loop all the transformations with its test cases.

Reply:
Done! No individual test cases failed for the modified transformations.